e2e/support/test_support.go

```diff
@@ -2685,7 +2685,44 @@ func CreateOperatorRoleBinding(t *testing.T, ctx context.Context, ns string) err
 	return nil
 }
 
-func CreateKamelPod(t *testing.T, ctx context.Context, ns string, name string, command ...string) error {
+func CreateKamelPodWithIntegrationSource(t *testing.T, ctx context.Context, ns string, name string, integration v1.ValueSource, command ...string) error {
+	// This line prevents controller-runtime from complaining about log.SetLogger never being called
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
I think this one should be something generic instead of specific to some given test.